OpenSim/Simulation/Model/Muscle.cpp

```diff
@@ -144,6 +157,7 @@ void Muscle::constructProperties()
     // By default the min and max controls on muscle are 0.0 and 1.0
     setMinControl(0.0);
     setMaxControl(1.0);
+    upd_GeometryPath().setDefaultColor(DefaultMuscleColor);
```

Comment:
I'm still confused. Will custom default colors for PathSprings be properly deserialized? For PathSpring, setDefaultColor is called in `extendFinalizeFromProperties()`, not in `constructProperties()`, so it seems the behavior would be different (that user-defined default colors would always be overridden as green).